### PR TITLE
// display the prefered modules popin

### PR DESCRIPTION
**Pull request by mickaelandrieu**

### CODE REVIEW

File: src/PrestaShopBundle/Controller/Admin/ModuleController.php [hunk 1]
```diff
@@ -253,16 +253,27 @@ public function notificationAction(Request $request)
         ));
     }
 
+    public function getPreferredModulesAction()
+    {
+        $controller = new \AdminModulesControllerCore();
+        ob_start();
+
+        $controller->ajaxProcessGetTabModulesList();
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Controller/Admin/ModuleController.php]: @tchauviere I'm pretty sure we can re-use actions from the actual Controller, but don't know which one :)

**Comment by mickaelandrieu** [comment]
I agree to rebase, if someone merge asap. Else I prefer wait until someone is available.

**Comment by Quetzacoalt91** [comment]
Still waiting !

**Comment by mickaelandrieu** [comment]
done ;)

**Comment by Quetzacoalt91** [comment]
cool_2

**Pull request merged**